vhdbuilder/packer/buildperformance/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/agentBaker/vhdbuilder/packer/build-performance/pkg/service"
+)
+
+func main() {
+	// Recover from panic and exit gracefully in order to prevent failing pipeline step
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("Recovered from panic in main:", r)
+			os.Exit(0)
+		}
+	}()
+
+	config, err := service.SetupConfig()
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("Program config set")
```

Comment:
maybe would be better to use the `log` package in this case?

Reply:
Will do

Reply:
Changed to using the log package throughout most of the program.